.github/workflows/prerelease.yml

```
@@ -0,0 +1,81 @@
+      - 'tests/**'
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    outputs:
+      is_push_to_main: ${{ steps.push_check.outputs.is_push_to_main }}
+      sha_short: ${{ steps.vars.outputs.sha_short }}
+
+    steps:
+    - uses: actions/checkout@v4
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)